sentry/processor.py

```diff
@@ -0,0 +1,137 @@
+""" Custom class of raven.core.processors taken of https://git.io/JITko
+    This is a custom class of processor to filter and sanitize
+    passwords and keys from request data, it does not exist in
+    sentry-sdk.
+"""
+
+
+import re
+
+from sentry_sdk._compat import text_type
+
+from .generalutils import string_types, varmap
+
+
+class SanitizeKeysProcessor:
+    """Class from raven for sanitize keys, cookies, etc
+    Asterisk out things that correspond to a configurable set of keys."""
+
+    MASK = "*" * 8
+
+    def process(self, data, **kwargs):
+        if "exception" in data:
+            if "values" in data["exception"]:
+                for value in data["exception"].get("values", []):
+                    if "stacktrace" in value:
+                        self.filter_stacktrace(value["stacktrace"])
+
+        if "request" in data:
+            self.filter_http(data["request"])
+
+        if "extra" in data:
+            data["extra"] = self.filter_extra(data["extra"])
+
+        if "level" in data:
+            data["level"] = self.filter_level(data["level"])
+
+        return data
+
+    @property
+    def sanitize_keys(self):
+        pass
```

Comment:
If you want an easy way to increase coverage, you can just edit `test_processor.py` and just add calls to the functions here that are marked as not covered, and run an `assert` on the result. You'd be testing trivial stuff, like, what if the input is None, what if the input is a `bytes` object and not a string, etc. It's like ordering `-1` beers, `1000000` beers etc. But it increases coverage.